### ISSUE TITLE
MiniProject1 Idea

### ISSUE THREAD

**Issue by BuddySwan** [elidberb/MiniProject1#1]
Hey Elijah, my idea is to find examples of opinion news articles about similar topics from different sides of the political spectrum (Fox vs. CNN, for example) and find the words that are commonly used. Ideally this will give an insight into the biases each side is using, by tracking the frequency of words that create an emotional response in the reader. Or words that tie an issue to a hot button topic certain readers will likely have a strong emotional reaction to.

**Comment by elidberb** [comment]
I like this idea. I'm curious to see what similarities/differences in words will arise.

**Comment by BuddySwan** [comment]
Thank you I am too. I have some theories about which words could be common but it would be interesting to have data to back up or disprove those theories

**Comment by elidberb** [comment]
If you are able to capture a list of names from both sides of the political spectrum, that could potentially show some interesting information. It might be outside the scope of this mini-project, but it could be interesting to see **who** the sources mention more.

**Comment by BuddySwan** [comment]
Yeah thats a good point. It could help show who each source is blaming or crediting for an issue. Or who the source is relying on for information
